### DISCUSSION TITLE
Park flag for vanilla/original ride value calculation.

### DISCUSSION THREAD

**Discussion by ZeeMaji** [OpenRCT2/OpenRCT2#21398] [Feature requests]
Something that's been on my mind for a while, but now that we have the new save format is there a chance we could see the currently custom build only vanilla/original ride value calculation option (RideRatingsCalculateValue's ORIGINAL_RATINGS in https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/openrct2/ride/RideRatings.cpp) turned into a park flag?

This has been bugging me for a long time, as it makes most flat rides and a few non-coaster tracked rides absolutely terrible especially in pay per ride scenarios to the point where they can barely even cover their own construction & operating costs over the course of most scenarios. I've begun to feel like this deviation from vanilla negatively impacts the game mainly in the original scenarios as it heavily discourages you from bothering with gentle rides and most thrill rides due to how they can barely make any money.

As for what this flag would be called, maybe you could make the OpenRCT2 price calculcations an option called "Harder ride values" and have the vanilla RCT2 calculations as the default? That seems pretty in line with the other "harder" park options like harder guest generation or harder park rating.

**Comment by MarcelVos96** [comment]
I like this idea very much as it allows for parity with the original while also keeping the OpenRCT2 way as an option. Perhaps the OpenRCT2 way could be altered to be harder in all situations, as right now it's only harder if you use rides with low ratings and even then only really in pay per ride scenarios.

**Comment by Nuesse97** [comment]
In addition to vanilla pricing, I would also like to have the possibility to use the vanilla calculation for park value.

**Comment by ZeeMaji** [comment]
Superseded by a total revert to the vanilla calculation in #25161, that and i also don't agree with this idea anymore since the now scrapped OpenRCT2 calculations didn't work well after thinking about them more, so i don't think an option to have them back is needed.
